src/Analyzer.Core.BuiltInRuleTests/Tests/AppService_EnableDiagLogs.json

```diff
@@ -0,0 +1,35 @@
+[
+    {
+        "Template": "AppServicesLogs", // didn't use the existing AppServices template because AppServiceWebApp_HTTPS would trigger for all the new resources, and AppService_EnableDiagLogs for all the already existing ones
+        "ReportedFailures": [
+            // The resources between lines 14 and 55 are not reported because their kind property includes "functionapp" or "linux"
```

Comment:
These comments were very helpful for me who is new to this, but other tests don't have this. Should this be standardized throughout all the tests? Or should a new property be added to the test framework that allows for these comments (and again standardized?)

Reply:
This PR made me like even more the idea we already discussed of having one template file per rule, which would make it super easy to document the passing test directly next to the resource

Reply:
These can also just be added as JSON properties if you want.  `"Description"` isn't actually part of the schema, but it doesn't hurt anything to have it defined.

Reply:
I'm not quite sure what you mean by documenting tests "next to the resource".

Reply:
I mean that if the templates were linked only to one rule we could add those description tags, or comments, next to each template resource that's meant to pass the analysis to explain why is part of the test